src/functionalTest/groovy/com/autonomousapps/android/projects/AndroidTestDependenciesProject.groovy

```diff
@@ -53,13 +53,13 @@ abstract class AndroidTestDependenciesProject extends AbstractProject {
           bs.android = AndroidBlock.defaultAndroidLibBlock(false)
           bs.dependencies = [commonsIO, commonsCollections, commonsMath, junit]
           // TODO update to support more versions of AGP
-          bs.additions = """\
-            androidComponents {
-              beforeUnitTests(selector().withBuildType("release")) {
-                enabled = false
-              }
-            }
-          """.stripIndent()
+//          bs.additions = """\
```

Comment:
My limited Android knowledge did not allow me to figure out why I needed to comment this out. If anyone has an idea, let me know.